cypress/e2e/inventory/holdings/check-new-formatting-statistical-code-on-holding-screen.cy.js

```diff
@@ -0,0 +1,125 @@
+import { LOCATION_NAMES } from '../../../support/constants';
+
+const testData = {
+  user: {},
+  item: {
+    instanceName: `Inventory Instance ${randomFourDigitNumber()}`,
+    itemBarcode: randomFourDigitNumber(),
+  },
+  statisticalCode: 'Book, print (books)',
+  secondStatisticalCode: 'Books, electronic (ebooks)',
+  calloutMessage: 'has been successfully saved.',
+};
+
+describe('Inventory', () => {
+  describe('Holdings', () => {
+    before('Create test data', () => {
+      cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+        testData.user = userProperties;
+        cy.getAdminToken()
+          .then(() => {
+            cy.getInstanceTypes({ limit: 1 });
+            cy.getInstanceIdentifierTypes({ limit: 1 });
+          })
+          .then(() => {
+            cy.createInstance({
+              instance: {
+                instanceTypeId: Cypress.env('instanceTypes')[0].id,
+                title: testData.item.instanceName,
+              },
+            }).then((specialInstanceId) => {
+              testData.item.instanceId = specialInstanceId;
+            });
+          });
+
+        cy.login(testData.user.username, testData.user.password, {
+          path: TopMenu.inventoryPath,
+          waiter: InventoryInstances.waitContentLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getHoldings({
+        limit: 1,
+        query: `"instanceId"="${testData.item.instanceId}"`,
+      })
+        .then((holdings) => {
+          testData.item.holdingUUID = holdings[0].id;
+        })
+        .then(() => {
+          cy.deleteHoldingRecordViaApi(testData.item.holdingUUID);
+        });
+      InventoryInstance.deleteInstanceViaApi(testData.item.instanceId);
+      Users.deleteViaApi(testData.user.userId);
+    });
+
+    it(
+      'C400653 Check the new formatting of Statistical codes field on Holdings create/edit screen (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        // 1 Go to "Inventory" application -> Search and select an Instance from precondition
+        InventoryInstance.searchByTitle(testData.item.instanceName);
+        InventorySearchAndFilter.verifyInstanceDisplayed(testData.item.instanceName);
+        InventorySearchAndFilter.checkRowsCount(1);
+        InventoryInstances.selectInstance();
+        InventoryInstance.waitLoading();
+
+        // 2 Click "Add holdings" button
+        InventoryInstance.pressAddHoldingsButton();
+        HoldingsRecordEdit.waitLoading();
+
+        // 3 Click "Add statistical code" button
+        HoldingsRecordEdit.clickAddStatisticalCode();
+
+        // 4 Click "Select code" dropdown" -> Check the dropdown
+        HoldingsRecordEdit.openStatisticalCodeDropdown();
+        HoldingsRecordEdit.verifyStatisticalCodeCount(1, false);
+        HoldingsRecordEdit.closeStatisticalCodeDropdown();
+
+        // 5 Click "Select code" dropdown" -> Check the dropdown
```

Comment:
comment is the same as in step 4, please change it

Reply:
done